.github/workflows/maestro_test.yml

```
@@ -0,0 +1,70 @@
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: gradle-${{ runner.os }}-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            gradle-${{ runner.os }}-
+
+      - name: Build Android Release APK
+        run: |
+          cd android
+          ./gradlew assembleExperimentalRelease --no-daemon --build-cache --parallel --max-workers=4
+          mv app/build/outputs/apk/experimental/release/app-experimental-release.apk app/app-experimental-release.apk
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)